pkg/query/fanin.go

```diff
@@ -1,188 +0,0 @@
-package query
```

Comment:
Why you moved that around? Easier to navigate through more, smaller files than single big one

Reply:
We had two weird small files, didn't feel right. Moved the iterator stuff out into its own file now